standard.tex

```diff
@@ -98,54 +103,42 @@ \subsection{PCB Design}
+\item Buy spares for all components provided it is not prohibitively expensive.
+\item All battery connectors should be terminated in a female connector to prevent battery shorting and enable easy battery swapping.
+\item Before ordering any components, make a list of all components in your design, and go through their data sheets one by one and ensure that they meet or exceed the requirements of your design. Do not just trust the Digikey table values, they are occasionally mislabeled. Design reviewers should also do this.
 \end{itemize}
-%end of mescellaneous subsection -- }}}
-%end of hard rules section -- }}}
+%-- }}}
+%-- }}}
 
-%start of documentation section -- {{{
+%Documentation -- {{{
 \section{Documentation Standards}
 It is your responsibility to ensure that your project is fully documented and available to the team. Development and documentation for all electrical projects should be tracked in git and hosted on GitHub (under the team organization, waterloo-rocketry). If you manage your repository using issues and pull requests, and if decisions regarding these are made in Slack conversations, put a permalink to the conversation in the issue comment (this is to help traceability). In general, how you manage your project repository is up to you.
 
-Release documentation for each project should also be put in Electrical/$\langle$project folder$\rangle$ on the Drive. Project documentation should include the following:
+Project documentation should include the following:
 
 \begin{enumerate}
-\item Users guide
+\item User's Guide/Checklist
 \begin{itemize}
 \item Step by step operating instructions.
-\item Partway through a procedure, if something can go wrong, say what it is, how you can tell it's gone wrong, and a reference to your troubleshooting guide on how to fix it.
-\item Assume that you will not be there when your system is used. Your documentation should be so good that someone who has never seen your system can fix/use it.
-\item This probably just needs to be a single checklist/procedure.
-\end{itemize}
-\item Troubleshooting Guide
-\begin{itemize}
-\item Include the symptoms of every failure mode, what the worst case scenario is, how to diagnose this failure mode conclusively, and how to fix it.
-\item This should have a list of every component in your system, and how to test that component in isolation. This list will be useful for building spares kits.
-\end{itemize}
-\item Design Documentation
-\begin{itemize}
-\item A document outlining the major decisions made for your project. The point of this is so that future generations maintaining your project know the reasoning behind your decisions. This can include meeting minutes, block diagrams, or anything else you think would be helpful. If it makes sense to reference slack conversations here, use a permalink to the relevant message.
```

Comment:
Why was this removed?

Reply:
Mostly cause none of the electrical projects this year did anything. We also have pretty high project turnover (though an argument could be made that projects that evolve from year to year should really have this). 

I'd be willing to keep it if the other people on this review are also willing, but if we're not gonna follow through I would rather remove it from the standard.

Reply:
"We haven't done documentation this year" isn't good justification for not mandating documentation, in my opinion. The appropriate action here seems to be for everyone to make documentation, not to shrug and give up on trying to document our projects.

At a bare minimum I think block diagrams/architecture description should be required. We don't need to write a report for everything but we should have adequately documented what each system does and how it does it.

Reply:
We do need to write a report actually. why don't we just specify what you need for the irec report, and we'll get that done early?

Reply:
@jacobdeery thoughts on this?

Reply:
Design Documentation

A document outlining the major decisions made for your project. There are two purposes for this: first, so that future generations maintaining your project know the reasoning behind your decisions; second, because writeup on all systems is required in the IREC report. The following information is needed for the report, but feel free to add any other documentation that you think will be helpful.

- Description of the system's purpose
- Relevant design constraints and criteria
- Block diagrams (and other diagrams, if you'd like)
- Description of the system's functionality
- Description of the relevant design decisions/tradeoffs involved in developing the system (dev logs are great)
- Descriptions of any analysis/testing performed on the system

All other required documentation (checklists, failure modes/troubleshooting) is described elsewhere.